normandy/classifier/api.py

```diff
@@ -6,7 +6,8 @@
 
 
 class FetchBundle(views.APIView):
-
+    authentication_classes = []
+    permission_classes = []
```

Comment:
We do, because otherwise the view requires a CSRF token due to the default auth/permissions.